### ISSUE TITLE
Explore requesting to add Grackle as an approved add-on

### ISSUE THREAD

**Issue by egamble23** [18F/delivery-assurance#151]
### Description
Building on ticket #87 of creating accessibility checklist for Google-based tools -- during desk research, an add-on tool named Grackle was found to be used by external organizations to do automated accessibility checks for Google-based tools. There is currently no built-in accessibility checking tool for Google tools. 

Grackle is not currently on the list of approved add-ons. This ticket would be to explore Grackle, look at how to gain review/approval for it as an add-on, and to socialize the tool for Google-based accessibility reviews. 

The Grackle add-on page is:
https://workspace.google.com/u/1/marketplace/app/grackle_docs/1085622905455

https://insite.gsa.gov/employee-resources/information-technology/it-help-desk-and-assistance/service-catalog/google-addon-request

### Point of contact on this issue
Posed by Erin Gamble, seconded by Jona Decker in the Accessibility Guild Slack channel #g-accessibility 

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- [x] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [x] No

**How much time do you anticipate this work taking?**
3-5 days 

### Acceptance Criteria

- [ ] What does "done" look like?
- [ ] Desk research on pros/cons for using Grackle
- [ ] Recommendation about submitting a request for it to be an approved add-on
- [ ] If Grackle request is submitted and completed, to socialize it as a tool including adding it as a link to the accessibility checklist for Google-based tools on #87 as well as sharing in the Accessibility Guild channel for future reference and use
